torch_xla/experimental/custom_kernel.py

```diff
@@ -8,15 +8,15 @@
 import torch_xla.core.xla_model as xm
 import torch_xla.distributed.spmd as xs
 
-from typing import Any, List, Callable
+from typing import Any, List, Callable, Optional
 from torch.library import impl
 from torch_xla.core.xla_model import XLA_LIB
 
 _XLA_USE_BF16 = os.environ.get("XLA_USE_BF16", "0") == "1"
 
 
 def _extract_backend_config(
-    module: "jaxlib.mlir._mlir_libs._mlir.ir.Module") -> str | None:
+    module: "jaxlib.mlir._mlir_libs._mlir.ir.Module") -> Optional[str]:
```

Comment:
I suspect this is some python version? I wonder why this only happens on upstream CI... Did you verified that with this fix upstream CI will pass?

Reply:
This will only report error if the python version is below 3.10 while Torch is testing with 3.8. Our CI is using `/usr/local/lib/python3.10/site-packages` so it's not an issue here. Also, the python syntax may not impact our docker build.